orchagent/neighorch.cpp

```diff
@@ -222,17 +243,16 @@ bool NeighOrch::addNextHop(const IpAddress &ipAddress, const string &alias)
     sai_status_t status = sai_next_hop_api->create_next_hop(&next_hop_id, gSwitchId, (uint32_t)next_hop_attrs.size(), next_hop_attrs.data());
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to create next hop %s on %s, rv:%d",
-                       ipAddress.to_string().c_str(), alias.c_str(), status);
+        SWSS_LOG_ERROR("Failed to create next hop %s, rv:%d",
+                       nexthop.to_string().c_str(), status);
         task_process_status handle_status = handleSaiCreateStatus(SAI_API_NEXT_HOP, status);
         if (handle_status != task_success)
         {
             return parseHandleSaiStatusFailure(handle_status);
         }
     }
 
-    SWSS_LOG_NOTICE("Created next hop %s on %s",
-                    ipAddress.to_string().c_str(), alias.c_str());
+    SWSS_LOG_NOTICE("Created next hop %s", nexthop.to_string().c_str());
```

Comment:
Please revert back to previous log.

Reply:
It is not appropriate to revert this log to previous, since it will not contain full NH description.

Reply:
@prsunny New NextHopKey::toLogString() function maintains the original output.

Reply:
@prsunny All logs are reverted to previous format.